.fixtures.yml

```diff
@@ -5,10 +5,7 @@ fixtures:
     apt:
       repo:           'git://github.com/puppetlabs/puppetlabs-apt'
       branch:         '1.8.x'
-    concat:
-      repo:           'git://github.com/puppetlabs/puppetlabs-concat'
-      branch:         '1.2.x'
-    file_concat:      'git://github.com/electrical/puppet-lib-file_concat'
```

Comment:
It was, but was dropped before 2.0.0 shipped during this refactor: https://github.com/puppetlabs/puppetlabs-concat/pull/306